### PR TITLE
SV-538 Update GetStandard to be by any Id

### PR DESCRIPTION
**Pull request by CodeWolf86**

### CODE REVIEW

File: src/SFA.DAS.Assessors.Api.UnitTests/Controllers/WhenGettingTrainingCourseByStandardUId.cs [hunk 1]
```diff
@@ -50,14 +50,14 @@ public class WhenGettingTrainingCourseByStandardUId
...
 
-            var controllerResult = await controller.GetByStandardUId(string.Empty) as BadRequestResult;
+            var controllerResult = await controller.GetStandardById(string.Empty) as BadRequestResult;
 
             controllerResult.StatusCode.Should().Be((int)HttpStatusCode.BadRequest);
         }
 
         [Test, MoqAutoData]
         public async Task And_Invalid_StandardUID_Then_Returns_Not_Found(
-            string standardUId,
```
Comment (DevenKShah) [inline src/SFA.DAS.Assessors.Api.UnitTests/Controllers/WhenGettingTrainingCourseByStandardUId.cs:60]: Method and class name has StandardUId, can be renamed.

File: src/SFA.DAS.SharedOuterApi.UnitTests/InnerApi/Requests/WhenBuildingTheGetStandardDetailsByStandardUIdRequest.cs [hunk 2]
```diff
@@ -8,9 +8,9 @@ namespace SFA.DAS.SharedOuterApi.UnitTests.InnerApi.Requests
     public class WhenBuildingTheGetStandardDetailsByStandardUIdRequest
     {
         [Test, AutoData]
-        public void Then_The_Url_Is_Correctly_Built(GetStandardDetailsByStandardUIdRequest actual)
+        public void Then_The_Url_Is_Correctly_Built(GetStandardDetailsByIdRequest actual)
```
Comment (DevenKShah) [inline src/SFA.DAS.SharedOuterApi.UnitTests/InnerApi/Requests/WhenBuildingTheGetStandardDetailsByStandardUIdRequest.cs:11]: Filename can be changed

**Review by DevenKShah — APPROVED** [review]
Couple of residues of StandardUId left out.

File: src/SFA.DAS.Assessors.Api.UnitTests/Controllers/WhenGettingTrainingCourseByStandardUId.cs (hunk 1 above)
Comment (CodeWolf86) [reply src/SFA.DAS.Assessors.Api.UnitTests/Controllers/WhenGettingTrainingCourseByStandardUId.cs:60]: Updated

File: src/SFA.DAS.SharedOuterApi.UnitTests/InnerApi/Requests/WhenBuildingTheGetStandardDetailsByStandardUIdRequest.cs (hunk 2 above)
Comment (CodeWolf86) [reply src/SFA.DAS.SharedOuterApi.UnitTests/InnerApi/Requests/WhenBuildingTheGetStandardDetailsByStandardUIdRequest.cs:11]: Updated

**Pull request merged**